buidl/mnemonic.py

```diff
@@ -98,6 +98,12 @@ def bytes_to_mnemonic(b, num_bits):
     return " ".join(mnemonic)
 
 
+def dice_rolls_to_mnemonic(dice_rolls):
+    """returns a mnemonic from a string of dice rolls (>=99 rolls recommended)"""
+    rolls_hash = sha256(dice_rolls.encode())
```

Comment:
Let's add a sanity check that `dice_rolls` is a string containing only the numbers 1-6. Anything else and it should throw an error.

Reply:
Agreed.  Added in 6de06301bbfc795e8474bca57d1d3b019c5c8da0.